homeassistant/components/tado/sensor.py

```diff
@@ -124,6 +125,30 @@ def get_geofencing_mode(data) -> str:
         attributes_fn=lambda data: {"time": data["weatherState"]["timestamp"]},
         data_category=SENSOR_DATA_CATEGORY_WEATHER,
     ),
+    TadoSensorEntityDescription(
+        key="freshness",
+        translation_key="freshness",
+        state_fn=lambda data: data["freshness"]["value"],
+        data_category=SENSOR_DATA_CATEGORY_AIRCOMFORT,
+    ),
+    TadoSensorEntityDescription(
+        key="last open window",
+        translation_key="last_open_window",
+        state_fn=lambda data: data["freshness"]["lastOpenWindow"],
+        data_category=SENSOR_DATA_CATEGORY_AIRCOMFORT,
+    ),
+    TadoSensorEntityDescription(
+        key="ac powered on",
+        translation_key="ac_powered_on",
+        state_fn=lambda data: data["freshness"]["acPoweredOn"],
+        data_category=SENSOR_DATA_CATEGORY_AIRCOMFORT,
+    ),
```

Comment:
Oh right, this is a binary sensor, we should move this one to the binary_sensor.py file, we can do that in another PR

Reply:
done

Reply:
done